src/www/ui/async/AjaxExplorer.php

```diff
@@ -419,8 +420,7 @@ private function createFileDataRow($child, $uploadId, $selectedAgentId, $pfileLi
     // override green/red flag with greenRed flag in case of single file with decision type "Do Not Use"
     $isDecisionDNU = $this->clearingDao->isDecisionDNU($childUploadTreeId, $groupId);
     $img = $isDecisionDNU ? 'redGreen' : $img;
-
-    return array($fileName, $licenseList, $editedLicenseList, $img, "$filesCleared/$filesToBeCleared", $fileListLinks);
+    return array($fileName, $licenseList, $editedLicenseList, $img, "$filesCleared/$filesToBeCleared/$totalFilesCount", $fileListLinks);
```

Comment:
Can we have spaces between the `/` like `1 / 4 / 6` ? Will it look good?

Reply:
I think @NicolasToussaint also mentioned that, but I was not sure if this is a good change. Is it?

Reply:
I've changed it as @GMishx proposed.